src/components/sign-up.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import logo from '../assets/logo.svg';
+import facebook from '../assets/facebook.svg';
+import gmail from '../assets/gmail.svg';
+export default function SignUp() {
+  return (
+    <div
+      class="mt-5"
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        maxWidth: '600px',
+      }}
+    >
+      <div
+        class="mb-5"
+        style={{ display: 'flex', justifyContent: 'space-around' }}
+      >
+        <img src={logo} alt="logo" />
+      </div>
+      <div
+        class="mb-5 pb-1"
+        style={{
+          width: '90%',
+          marginLeft: 'Auto',
```

Comment:
it should be auto, and both this and the next line can be replaced with `mx-auto` class